commands/sintercard.md

```diff
@@ -1,8 +1,13 @@
+This command is similar to `SINTER`, but has a completely different syntax.
```

Comment:
Why add this? `SINTER` and `SINTERCARD` are very different command and also have a different syntax. I think we shouldn't add this.

Reply:
`ZINTERCARD` also mention `ZINTER`. My first thought was to add something like this.. 
> This command is similar to `ZINTER`, but instead of returning the result set, it returns just the cardinality of the result.